.pax/post-packaging.sh

```diff
@@ -14,6 +15,10 @@ set -x
 SCRIPT_NAME=$(basename "$0")
 CURR_PWD=$(pwd)
 
+#
+# SCRIPT ENDS ON FIRST NON-ZERO RC
+#
+
```

Comment:
This is what -e does

Reply:
Thanks for the info. 
Problem with this is taht we do not clean up SMPE build datasets if the build fails.
I'll tinker with smpe.sh to do the cleanup there instead, and push that with another PR.